core/src/tile/tileManager.h

```diff
@@ -32,11 +32,7 @@ class TileManager {
 
 public:
 
-    /* Returns the single instance of the TileManager */
-    static std::unique_ptr<TileManager> GetInstance() {
```

Comment:
And would it still be useful? This implementation does create an instance and then directly move it out as return value, so each call to GetInstance returns a new instance.